src/common/updates.cpp

```diff
@@ -96,7 +96,7 @@ namespace tools
 
   std::string get_update_url(const std::string &software, const std::string &subdir, const std::string &buildtag, const std::string &version)
   {
-    static const char base[] = "https://updates.getmonero.org/";
+    static const char base[] = "http://updates.getmonero.org/";
```

Comment:
@anonimal we'd have to add an entire library in to handle the automated download via HTTPS. Instead of doing that, we take the quick win now (which can just use epee), and rather add in a library that lets nodes download *and* serve the update via bittorrent:)